### DISCUSSION TITLE
Rename Joomla! to Joomla

### DISCUSSION THREAD

**Discussion by PhocaCz** [joomla/joomla-cms#35730] [General]
If there is already a discussion on this, it would be good to see a link. If not, what's stopping the renaming of "Joomla!" to "Joomla".

Every time I write something about Joomla!, I run into consistency problems:

- The correct word should be Joomla! but we usually see another version

![img](https://i.imgur.com/RIWIqjf.png)

![img](https://i.imgur.com/33LMDYI.png)

- We can't actually use the first option at the end of a sentence or in a question, etc.

![img](https://i.imgur.com/3blEaUE.png)

- When using different alternatives, I do not see uniform rules
- Some languages use inflection, there again the exclamation mark is useless
- There's nothing worse than product name inconsistency
- Etc.

I think the exclamation mark at the end of the name is quite unfortunate, and the inconsistency of the name certainly doesn't help the project.

**Comment by brianteeman** [comment]
the registered trademark is for "JOOMLA!"

interface guidelines https://developer.joomla.org/en-gb-user-interface-text-guidelines/joomla-name.html

**Comment by PhocaCz** [comment]
Thank you for the link. So not having the "Bang!" at the end would significantly shorten this guide as well.

**Comment by sozzled** [comment]
This discussion is probably over before it even began and it doesn't really matter what I think.  Yes, I agree with @PhocaCz that someone should take a mighty big stick to the people who wrote Joomla (without the exclamation mark) on all those articles that appear at joomla.org and they should beat the living daylights out of them for not writing "Joomla!" consistently.  Yes, naughty, naughty **_bad_** people!  Shame on them.  Someone should rewrite all the documentation, the magazine articles, everything in the Extensions Directory, the Volunteers Portal, at opensourcematters.org or wherever you can find the inconsistent use of Joomla instead of Joomla! and fix every abused instance.  Or spend a few tens of thousands of dollars to hire an army of editors to do that job.

And when I spent a few weeks editing/rewriting the Wikipedia article, I may have used Joomla/Joomla! inconsistently, too.

But, to be serious, none of this is going to happen in my lifetime.  @brianteeman is right:  Joomla! (with the exclamation point) is the legal trademark owned by Open Source Matters, Inc and registered in the USA; the correct spelling is Joomla! (with the exclamation point).  I try to spell things correctly; sometimes I make mistakes.   I'm human.  So were the people who used Joomla/Joomla! inconsistently.  Give it a rest; let it go.  It's too late to call the grammar police.

**Comment by PhocaCz** [comment]
> Joomla! (with the exclamation point) is the legal trademark owned by Open Source Matters, Inc and registered in the USA; 

So this in fact means, that "Joomla" is not a trademark and in fact not protected.

BTW In this case, I don't care about grammar, or the annoyance of someone who doesn't use an exclamation point. It's the other way around. The rules for using both variants are so complicated that we can't be surprised if someone doesn't follow them. Because although only the exclamation mark variant is protected, in some cases the non-exclamation mark variant is recommended, and in some cases even grammar won't allow the exclamation mark. And that's where I see the problem. Having a protected name where the grammar doesn't allow you to use it is not a happy name choice.

**Reply by sozzled** [reply]
>  ... in some cases the non-exclamation mark variant is recommended ...

Who says that?  Where is this recommended?

> ... in some cases even grammar won't allow the exclamation mark ...

Huh?  (or should I have written "Huh?!"?).  What cases are these?

**Comment by PhocaCz** [comment]
See this guide:
https://developer.joomla.org/en-gb-user-interface-text-guidelines/joomla-name.html

There is written where to not use "!".

Grammar does not allow something like this:

It is Joomla!. Is it Joomla!?

Right is:
It is Joomla! or: It is Joomla. And: Is it Joomla?

And this is the problem when grammar or other rule changes your name to more variants.

**Comment by sozzled** [comment]
Fair enough.  I understand what you're saying.  The style guide is confusing.  A lot of things in this world are confusing.  I don't care how people write, spell, pronounce or even call J! (as long as people don't call J! something like Wordpress).  If people want to write Joomla! as "Joomla", "J!", "Joombla", "Jumla", we all know what they're saying.

You may also be interested in the recommended citation of the J! trademark: https://docs.joomla.org/Joomla:Trademarks.  I confess that I haven't always strictly adhered to this style on the one or two websites where I write about J!.  Perhaps, one day, I may tidy those things up.  Many people don't even bother with disclaimers (like the following one) on their websites:

"_[Your site name]_ or _[Your business name]_ is not affiliated with or endorsed by The Joomla! Project™  or Open Source Matters (OSM). The Joomla!® name and logo is used under a limited license granted by Open Source Matters Inc. the trademark holder in the United States and other countries."

So what?

If we extend this discussion to the "inconsistent" use of J! in language constants, who knows where this will end?

**Comment by PhocaCz** [comment]
> Many people don't even bother with disclaimers (like the following one) on their websites

In fact, if they use "Joomla" then they don't need to include disclaimers, because "Joomla" is not protected. Only in case, they use "Joomla!".

**Comment by PhocaCz** [comment]
But I started this discussion because I by myself don't like to think about which is the right form in specific cases and don't like the inconsistency. In my opinion, the inconsistency hurts the name.

**Reply by sozzled** [reply]
If that's the case, any time someone uses the word "Joomla!" (with the bang) on their website(s) and they don't add the appropriate disclaimer, they're liable to OSM taking legal action if OSM believes the name has been misappropriated. I don't think OSM would be worried about what one guy, whose insignificant thoughts about J! and who lives on the arse-end of the planet, has to say about J! in any event or whether to send the school disciplinarian out to give me a spanking for omitting the disclaimer.

As far as I'm concerned, if you're concerned about any ambiguity, take the matter up with the Trademark team: https://tm.joomla.org/contact

**Comment by PhocaCz** [comment]
> As far as I'm concerned, if you're concerned about any ambiguity, take the matter up with the Trademark team

This discussion is not a priori about trademark, it is about use of unique name everywhere.

**Reply by sozzled** [reply]
It's simple: "Joomla!" is a trademark (as @brianteeman wrote earlier); "Joomla" is not.

In theory, if you use the **_trademarked name_** anywhere on your website, you should add the appropriate disclaimer somewhere.

**Comment by PhocaCz** [comment]
> In theory, if you use the trademarked name anywhere on your website, you should add the appropriate disclaimer somewhere.

As written above, this is not the subject of this discussion. This discussion is about the interest of the Joomla! project. The question is whether it is better to use unique name "Joomla" for this project rather than different names in different places "Joomla" vs "Joomla!". It's not about the current state, but about the possible future state (regardless of whether any name will be trademarked). The purpose of this discussion is whether not to use unique name everywhere.
